tests/integration/bond_test.py

```diff
@@ -265,3 +265,40 @@ def test_set_bond_mac_address(eth1_up):
         eth1_cur_state = current_state[INTERFACES][1]
         assert bond99_cur_state[Interface.MAC] == MAC1.upper()
         assert bond99_cur_state[Interface.MAC] == eth1_cur_state[Interface.MAC]
+
+
+@pytest.fixture
+def bond99():
+    with bond_interface(name='bond99', slaves=['eth1', 'eth2']) as bond_state:
+        yield bond_state
+
+
+def test_reordering_the_slaves_does_not_change_the_mac(bond99):
+    # Adding slaves to the bond
```

Comment:
You can drop the comments, I consider comments the last resort.
If you need a comment, it means the code is not clear enough: Merits extracting to a function that expresses what the comment said.
In this case, I think it is pretty clear, but you can do it if you want.